### DISCUSSION TITLE
some Classes are not applyied at all

### DISCUSSION THREAD

**Discussion by ObaidAshiq** [tailwindlabs/tailwindcss#12776] [Help] [answered]
As u can see in the image i have used w-[340px] ,
but itsnt applied at all
I am using next js 
![image](https://github.com/tailwindlabs/tailwindcss/assets/133643704/fce3e6b6-0bc7-41f1-9df6-ffb7a84a8ac9)
`/** @type {import('tailwindcss').Config} */
module.exports = {
  important: true,
  content: [
    "./src/**/*.{js,ts,jsx,tsx,mdx}",
    // "./pages/**/*.{js,ts,jsx,tsx,mdx}",
    "./Components/**/*.{js,ts,jsx,tsx,mdx}",
    
    // "./app/**/*.{js,ts,jsx,tsx}",
    // Or if using `src` directory:
  ],
  theme: {
      extend: {
  
        fontFamily: {
          'sans': ['ui-sans-serif', 'system-ui', 'Inter']
          
        },

        screens: {
          'msm': {'max' : '639px'},
          'mxsm': {'max' : '420px'},
          
       },
  
        colors: {
          success50:"#ECFDF3",
          success700:"#027A48",
          gray: {
            25: "#FCFCFD",
            50: "#F9FAFB",
            100: "#F2F4F7",
            200: "#EAECF0",
            300: "#D0D5DD",
            400: "#98A2B3",
            500: "#667085",
            600: "#475467",
            700: "#344054",
            800: "#1D2939",
            900: "#101828",
          },
          
          primary: {
            25: "#FCFAFF",
            50: "#F9F5FF",
            100: "#F4EBFF",
            200: "#E9D7FE",
            300: "#D6BBFB",
            400: "#B692F6",
            500: "#9E77ED",
            600: "#7F56D9",
            700: "#6941C6",
            800: "#53389E",
            900: "#42307D",
          },
          ...some more colours
      
      },
  
      },
    
    
  },
  plugins: [require('@tailwindcss/container-queries'),require('tailwind-scrollbar')({ nocompatible: true }),require('@tailwindcss/forms')],
  variants: {
    scrollbar: ['rounded']
}
}`

**Comment by wongjn** [comment, marked as answer]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).

**Comment by ObaidAshiq** [comment]
There more thing that i noticed
when i write width to be 400 px and i write some other value than 200px it doesnt show up 
![image](https://github.com/tailwindlabs/tailwindcss/assets/133643704/5f266f53-ca3b-4422-84ab-7c1421001227)
but when i write 200px , it works
![image](https://github.com/tailwindlabs/tailwindcss/assets/133643704/aba615d3-d507-4dea-a144-274e22968495)
Can someone explain what is happening

**Reply by wongjn** [reply]
It seems like perhaps the `content` file globs do not sufficiently cover the file that the code appears in or
 you may be using [string concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). In either case, the `w-[400px]` is not seen by Tailwind at build time so the CSS rule for `w-[400px]` is not generated. `w-[200px]` works because the string `w-[200px]` appears in another file that *is* covered by the `content` file globs.
